session_0/word_processing.rb

```diff
@@ -2,13 +2,25 @@
 # any non-lower case alphabet (A..Z) to corresponding lower case
 # alphabet
 def lower_case(words)
-  raise NotImplementedError # TODO
+  output = Array.new(words.length)
+  i = 0
+  l = words.length
+  while i < l do
```

Comment:
try using the map construct instead
https://www.rubyguides.com/2018/10/ruby-map-method/